lab-justin/test/greet-test.js

```diff
@@ -0,0 +1,13 @@
+'use-strict';
+
+const greet = require(__dirname + '/../lib/greet');
+const assert = require('assert');
+
+describe('test module lib/index.js', function(){
+  describe('test function index()', function(){
+    it('should return HELL-O' , function(){
```

Comment:
Keep the descriptions in the it blocks consistent with what is actually being tested.